### ISSUE TITLE
Suggestion: Support Partial Scrolling through Arrow Keys

### ISSUE THREAD

**Issue by crazyuploader** [desktop/desktop#9074]
### Describe the feature or problem you’d like to solve

Support Scrolling Changes of any file with Arrow Keys.

### Proposed solution

How will it benefit Desktop and its users? It can help people like with broken scroll wheel to scroll a little, pressing page down works but that's too much to scroll. Perhaps, giving to scroll line-by-line would make more sense while reviewing changes before committing them? 

### Additional context

Add any other context like screenshots or mock ups are helpful, if applicable.
Modify any file, select any modified file, press up/down arrow keys to see 'em not working at all.

Thanks in advance ;)

**Comment by niik** [comment]
Hey @crazyuploader, thanks for opening this! I totally agree that you should be able to scroll the diff using the up/down keys. I was surprised to find that wasn't the case to be honest and realized after looking at it that it's because when you're using Up/Down you're actually moving an invisible cursor inside of the diff itself.

We use an editor (CodeMirror) to render the diff but we're using it in read-only mode so the cursor is hidden but it's still the means by which CodeMirror navigates around the diff.

You can see this by pressing (and holding) the Down key for a while. Eventually the diff will start scrolling as the invisible cursor makes it down to the last line of the visible document.

I think we should fix this and I don't believe it'll be all that hard to do but in the meantime at least you'll have a workaround.

**Comment by crazyuploader** [comment]
Thanks @niik, I hope you got my point, since as an end user I'd like to relax, sip on my tea while reviewing my changes that I've done without caring if it got an invisible cursor or not, since for me the cursor doesn't even exist, so it'd make much more sense if the scroll starts instantly, not when the invisible cursor reaches the end of the screen. 

Thanks, for your help.

**Comment by niik** [comment]
> Thanks @niik, I hope you got my point, since as an end user I'd like to relax, sip on my tea while reviewing my changes that I've done without caring if it got an invisible cursor or not, since for me the cursor doesn't even exist, so it'd make much more sense if the scroll starts instantly, not when the invisible cursor reaches the end of the screen.

Hey @crazyuploader, yup I hear you. I've opened https://github.com/desktop/desktop/pull/9085 which I'm hoping will ship in the next major release of GitHub Desktop. My explanation about the invisible cursor was only meant as a temporary workaround until we're able to address the problem properly.
